packages/common/src/services/explore/Explore.ts

```diff
@@ -83,28 +82,29 @@ export class Explore {
       )
       if (!lineupItems.length) return []
 
-      const tracks = lineupItems.filter(
-        (lineupItem): lineupItem is UserTrack => 'track_id' in lineupItem
-      )
-      const history = await sdk.full.users.getUsersTrackHistory({
-        id: encodeHashId(currentUserId),
-        limit: 100
-      })
-      const activityData = history.data as APIActivityV2[]
-      const listenedToTracks = activityData
-        .map(makeActivity)
-        .filter(removeNullable) as UserTrackMetadata[]
-
-      // Imperfect solution. Ideally we use an endpoint that gives us true/false
-      // if a user has listened to a passed in array of tracks.
-      const listenendToTrackIds = listenedToTracks.map(
-        (track) => track.track_id
-      )
-
-      const notListenedToTracks = tracks.filter(
-        (track) => !listenendToTrackIds[track.track_id]
-      )
-      return notListenedToTracks.slice(0, limit)
+      throw new Error('Not implemented')
```

Comment:
wait isn't this one already using sdk?

Reply:
the `makeActivity` code uses stuff in apiClient that I removed because that still had the dep. It can be added back, but should be done as a separate util rather than relying on libs discovery node selector